app/sign-in/[[...sign-in]]/page.tsx

```diff
@@ -1,7 +1,7 @@
 import { SignIn } from '@clerk/nextjs';
 import { Box } from '@mantine/core';
 import { Metadata } from 'next';
-import { redirect } from 'next/navigation'; // Import redirect from next/navigation
+import { redirect } from 'next/navigation';
```

Comment:
is redirect used here specifically because we're calling on the server for `login` ?

Reply:
Good question, we don't use it it's showing "declared but never read" so we can probably remove this.

Reply:
interesting, I was reading the docs and it didnt really understand it.